hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HoodieHiveClient.java

```diff
@@ -426,7 +415,7 @@ public CommandProcessorResponse updateHiveSQLUsingHiveDriver(String sql) {
   private void createHiveConnection() {
     if (connection == null) {
       try {
-        Class.forName(HiveDriver.class.getCanonicalName());
+        Class.forName("org.apache.hive.jdbc.HiveDriver");
       } catch (ClassNotFoundException e) {
```

Comment:
Hi @pengzhiwei2018, thanks for your contribution.
IIUC,  when you are using this feature, no matter you set `HIVE_USE_JDBC_OPT_KEY` to true or false, the `hive-jdbc` jar has already been included in hudi binary jar. so this pr is unnecessary.

WDYT?

Reply:
Thanks for you response  @wangxianghu . When we debug code with the `HIVE_USE_JDBC_OPT_KEY=false`  in IDE, we just include the **hudi-spark** dependency,the hive-jdbc is no need for us. More dependencies include may lead to more conflicts. And also IMO it is better to follow the minimization principle.If we do not use the hive jdbc, we should not include the dependency in the code.Even though the jdbc has packaged in the `hudi-spark-bundle`.

Reply:
Sounds reasonable.